site/docs/compositemark.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: docs
+menu: docs
+title: Composite Mark
+permalink: /docs/compositemark.html
+---
+
+Composite marks are "macros" for more complex layered mark specs. Layered mark specs create visualizations with a series of basic mark types.
+
+| Property      | Type          | Description    |
+| :------------ |:-------------:| :------------- |
+| `mark`      | String        | The CompositeMark type. Supported composite marks include: "box-plot". |
+
+## Box Plot
+
+`box-plot` composite mark represents a box-plot or box and whisker plot. The lower and upper whiskers extend to the min and max respectively. The middle tick in the box represents the median. The lower and upper part of the box represents quartile 1 and 3 respectively.
+
+`box-plot` supports the following properties: color, size, opacity.
+
+### Vertical 1D Boxplot
```

Comment:
Need more narration.  

- What makes a boxplot 1D horizontal? (have 1 quantitative field on x) - same for vertical.  (maybe group them into a section about 1D boxplot

- Maybe group 2D boxplot into one section and describe what's the rule for determining orientation and mention that discrete x discrete / continuous x continuous do not make sense.

- Show example for customizing size and color

- Add a section for box plot config.
  - config.box.size
  - Talk about generated roles (give one full output) and how users can use config.<role>.*